mythplugins/mythmusic/mythmusic/visualize.cpp

```diff
@@ -937,7 +1175,7 @@ bool Spectrum::process(VisualNode *node)
     for (auto k = i; k < FFTW_N; k++)
     {
         m_dftL[k] = (FFTComplex){ .re = 0, .im = 0 };
-        m_dftL[k] = (FFTComplex){ .re = 0, .im = 0 };
+        m_dftR[k] = (FFTComplex){ .re = 0, .im = 0 };
     }
```

Comment:
I started to fix bugs in Spectrum based on what I learned in Spectrogram.  I'm sure that Spectrum is not right since it is not processing the whole signal.  But I will submit those fixes in a separate pull request.